alpha/apps/kaltura/lib/batch2/kFlowHelper.php

```diff
@@ -2967,6 +2967,49 @@ public static function addDeliveryTagToEntry(entry $entry)
 			$entry->save();
 		}
 	}
+
+	public static function handleDelayedNotification($object)
+	{
+		$delayedLockedJobs = [];
+		switch (true)
+		{
+			case ($object instanceof entry):
```

Comment:
why not just if condition ?

Reply:
maybe in the future we will have more conditions concerning other objects, not just entries